api/model/src/main/java/org/projectnessie/model/Operation.java

```diff
@@ -82,6 +105,15 @@ static Put of(ContentKey key, Content content, Content expectedContent) {
     }
   }
 
+  @Schema(
+      type = SchemaType.OBJECT,
+      title = "Delete-'Content'-operation for a 'ContentKey'.",
+      description =
+          "Used to delete an existing content key.\n"
+              + "\n"
+              + "If the key for a content shall change (aka a rename), then use a `Delete` "
+              + "operation using the current (old) key and a `Put` operation using the new key "
+              + "populating both the `value` and `expectedValue` fields. See `Put` operation.")
```

Comment:
The `populating both the `value` and `expectedValue` fields` part seem outdated now that `expected value` is deprecated?